### PR TITLE
Add ExposureConfiguration as new field in ExposureNotificationRequest

### PR DESCRIPTION
**Pull request by rayning0**
- Change ScoreV1() to use values from ExposureConfiguration, not hardcoded vals
- Add exposureconfig.json input to all tests for scoring service

### CODE REVIEW

File: model/scoring.go [hunk 1]
```diff
@@ -77,20 +77,21 @@ func FilterExposuresByDate(exposureSummaries *[]ExposureSummary, date int) *[]Ex
...
 	if request.NewExposureSummary.MatchedKeyCount == 0 {
-		return empty_response, errors.New("Matched key count was 0.")
+		return emptyResponse, errors.New("Matched key count was 0.")
 	}
 
-	weightedDuration := WeightedDuration(&request.NewExposureSummary)
+	weightedDuration := WeightedDuration(&request.NewExposureSummary, attenuationWeights)
 
 	if request.NewExposureSummary.MatchedKeyCount == 1 {
 		// TODO: Use config here.
```
Comment (lmartensson) [inline model/scoring.go:90]: Remove comment.

Comment (lmartensson) [inline model/scoring.go:90]: Remove this comment.

**Review by lmartensson — COMMENTED** [review]
Looks good, although don't submit this until the app has been updated to send the config.

**Comment by dvbnrg** [comment]
Looks good on my end as well. Nice job on the reorganization of the files and folders

**Pull request merged**